test/test_endpoints.py

```diff
@@ -350,3 +381,147 @@ def test_get_transaction_trace_of_rejected():
     resp_body = resp.json()
     assert resp_body["code"] == str(StarknetErrorCode.NO_TRACE)
     assert resp.status_code == 500
+
+
+@devnet_in_background()
+def test_send_message_to_l2_deploy_execute():
```

Comment:
Just now I realized that these tests are all in the unfortunate wildcard file (test_endpoints). I would move these new tests to a new file (something like `test_l1_l2_mock_messaging.py`).

Reply:
ok I'll do it at the end

Reply:
https://github.com/Shard-Labs/starknet-devnet/pull/365/commits/6d8fdc50f67c4cd944e5dd60e3b9f6197a96ff20